app/src/main/java/com/wesleyfranks/todo24/data/TodoAdapter.kt

```diff
@@ -11,6 +11,7 @@ import androidx.recyclerview.widget.ListAdapter
 import androidx.recyclerview.widget.RecyclerView
 import com.afollestad.materialdialogs.MaterialDialog
 import com.google.android.material.snackbar.Snackbar
+import com.wesleyfranks.todo24.R
```

Comment:
8/14 imports aren't being used. You should definitely go through all your files and remove the unused imports. You can:
1. Press "Shift Shift", and type "Optimize Imports" and the action will come up, or...
2. Press control + option + 'O'